incubator/hnc/test/e2e/acting-as.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "sigs.k8s.io/multi-tenancy/incubator/hnc/pkg/testutils"
+)
+
+var _ = Describe("Acting-as", func() {
+	const (
+		parent = "parent"
+		child  = "child"
+	)
+
+	BeforeEach(func() {
+		CleanupNamespaces(parent, child)
+	})
+
+	AfterEach(func() {
+		CleanupNamespaces(parent, child)
+	})
+
+	It("should allow acting as different service accounts", func() {
```

Comment:
What we're really testing here is that HNC will _reject_ calls if your service account has the wrong permissions. The "act-as" is just a feature _of the test_ that we use to test this.

I'd suggest something like `It("should reject hierarchy changes for users without the correct permissions")`.